src/create-menu/createMenu.js

```diff
@@ -299,6 +398,23 @@ function validateOptionsToWhatsappMenu(props, maxNumberOfOptions) {
     }
 }
 
+function getNumberOfOptions(menuOptions) {
+    if (typeof menuOptions === 'object') {
```

Comment:
Here you can invert the order of the code and put your if statement inside the try block, like:

```js
let optionsCount = 0;
try {
    if (typeof menuOptions === 'object') {
        // your logic here
    }
} finally {
    return optionsCount;
}
```

In my opinion it would make the code cleaner, what do you think?

Reply:
I agree. I changed it.